### ISSUE TITLE
[User testing] Combined Profile and Account: Create report and tickets

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#2588]
Epic #2092

[CPAA Research Findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Combine%20Profile%20and%20Account/Research/cpaa-research-findings.md)

- [x] Write research report
- [x] Create any tickets that come out of research

**Comment by Samara-Strauss** [comment]
Great job on the report, @tressaellen! Here are my comments

- **Research Goals**: Nitpicky punctuation details — can you please end the research questions statements (eg. "does the user feel like...?") with question marks? And for the hypotheses, can you add periods to the last two?
- **We used Zoom to conduct remote moderated usability testing sessions to observe how participants use profile 2.0.** — Can we just call it "the new profile" or something? I realize this is totally my fault for calling it profile 2.0, but that's more of a name for the project than the actual page for itself. Once the new design launches, we'll most definitely just call it the "profile." I saw other mentions of "profile 2.0" in the doc, so can you go through and remove those references? Thanks!
- **Participants preferred the new profile design to the old design, and easily found what they were looking for within the profile using the new navigation.** I would actually flip how this is worded to something like "People easily found what they were looking for within the new profile. Consequently, most participants preferred the new profile to the old one." It feels like the improved wayfinding (plus the visual design) lead to the preference for the new profile, so I don't want that detail to be lost.
- **The participant that preferred the old profile was in the 55-65 age range.** This detail doesn't feel relevant to me. Though unintentional, it feels like it's implying that they had this preference because of their age. I would either take this out or clarify what you're trying to say.
- **Participants had a difficult time finding the 'profile menu', and in general were confused about the mobile navigation.** Do you think a screenshot might help clarify the difference between the three menus you're talk about here? In general, are there visuals of the new design you think it could be helpful to include in the report for context or clarification about specific tasks? If nothing else, I would definitely link to the prototype so people who haven't seen it before can check out the updates.
- **Benefit Rating/entitlement (?)** Hmmm, is there any more context here? This person could have either been talking about their disability rating or generally what benefits they were eligible for. Not sure if there are additional context clues that might clarify.
- **Participants didn't notice the 'more information' drop down and even after reading the 'More Information' content there was confusion about updating the contact email vs the log-in email.** Can you add the following commas — "Participants didn't notice the 'more information' drop down, and even after reading the 'More Information' content, there was confusion about updating the contact email vs the log-in email."
- **Clarifier: Inn order for users to disconnect, they must first connect and that is done on a third-party app. The users we tested with did not have this real life context, when they gave their feedback.** Cleaning this up – "Note: Before a user can disconnect a third-party app, they have to connect it, but this can only be done via the app itself. Our test participants did not have this context."
- **Additional insights** Question: How did you differentiate between what should be in the key findings vs. the additional insights section? 
- **Additional insights and recommendations**  Can you please add periods to these statements?
- **Next steps**  Can you add something here about how we should further explore the placement of benefit information in the logged-in homepage discovery? 
- **Put together deck with proposals for current solution and future solutions (keeping logged-in homepage in mind)** Who is this deck for? Is this for VSP?

**Comment by tressaellen** [comment]
Thanks @Samara-Strauss ! I'll fix all the punctuation stuff (thank for looking over this - punctuation is not my strength). I'll also see if I can add context to the benefit rating/entitlement from the sessions.

- **Additional insights vs key findings** : For key findings - I started with 'Participants preferred the new profile design to the old design, and easily found what they were looking for within the profile using the new navigation.' because that addressed two of the requirements in the product outline (prioritize wayfinding and effectively use visual space).  The last requirement (scale gracefully) was not tested so I didn't include it. Should I reference that somewhere still? I chose the rest of the items because are the ones that will impact the future design/structure of the profile.  I felt that those should have the most documentation so validate these changes.

**Comment by Samara-Strauss** [comment]
Thanks for the clarification!  A couple things:

> The last requirement (scale gracefully) was not tested so I didn't include it. Should I reference that somewhere still?

Do you think we could extrapolate people's ability to easily navigate/their preference for the desktop navigation/design as support for the design's ability to scale (at least navigation-wise)? That may be making too many leaps, so definitely let me know what you think. We don't need to address this point if we don't feel like we have the evidence to support it.

>  I chose the rest of the items because are the ones that will impact the future design/structure of the profile. 

Got it. Makes total sense. In this case, I would think "Users liked the 'name-tag' area, but it's taking up too much space in the mobile view." could be a key finding because we have a directly actionable next step, but I'll leave that final call up to you.

**Comment by tressaellen** [comment]
Ooh, yes I think we could possibly put some language around the desktop navigation/design as support for the design's ability to scale... i'll try a few things and if it feels like too much of a stretch I'll leave it out. 

Also, good point about the name-tag - I'll bump that up and I have some great quotes to support that as well.

I also realized I never answered:

> " Put together deck with proposals for current solution and future solutions (keeping logged-in homepage in mind) Who is this deck for? Is this for VSP? "

Since this spans across multiple teams - the thought here is that I wanted to be able to communicate a need for a more long-term solution for the mobile navigation using whatever I can gather from other teams.  This could be done with a markdown file as well, I just want a place where I can keep all of the mobile navigation things documented and easily 'handoff-able' if needed.

**Comment by Samara-Strauss** [comment]
> Since this spans across multiple teams - the thought here is that I wanted to be able to communicate a need for a more long-term solution for the mobile navigation using whatever I can gather from other teams. This could be done with a markdown file as well, I just want a place where I can keep all of the mobile navigation things documented and easily 'handoff-able' if needed.

Got it. Makes total sense. Thanks for letting me know!

Everything else sounds good, too. Looking forward to seeing these updates when you get back. Have a great weekend!!

**Comment by tressaellen** [comment]
Hi @Samara-Strauss 
I've made all the updates to the research report, and it's linked in this ticket. One thing - it felt like too big of a stretch to include 'desktop navigation/design as support for the design's ability to scale' as it's own key finding, so I added a line in under number 2. The side nav increased scanability of the overall profile, especially in the desktop view.

It says, "It can be assumed that adding in new page links to the navigation would still be consistent with these findings and would support the capacity for the design to scale."

Thanks!

**Comment by Samara-Strauss** [comment]
@tressaellen noted on everything above. 

Also, the report is looking awesome! Just a few, small questions/comments below:

> Permanent vs Temporary
> Benefit entitlement
> Percentage Used

What are these referring to? By their name alone, I am not sure what this means re: veteran info/benefits. Can you clarify?

> FAQ's were noticed and utilized throughout the sessions - with exception to the email address FAQ.

I would mention that these are in-context FAQs since this could be confused with an FAQ page.

> First, there should be a short term fix specific to logged-in experience, followed by a more involved long term solution based on previous research and more comprehensive mobile navigation needs.

Totally agree here, but I might say "specific to the profile" instead of "specific to the logged-in experience" because the only context we have now for a new/different pattern is the profile.

> Possibly add benefits overview to profile (this hinges on decisions around the logged-in homepage)

I'd change this to something like "Further investigate what benefit information we should add to the profile." Basically, we should do some more research here as we pursue a logged inhomepage and navigation.

> Remove/shrink military badge in 'nametag'

Is this for mobile only? If so, I would mention that, unless you are thinking about changing it on desktop as well.

**Comment by tressaellen** [comment]
Thanks @Samara-Strauss, I've made all those changes. For re: veteran info/benefits - I went back through the transcripts and found some clarification. I ended up removing them from the list and adding 'Current benefits available/used'. Clarifiers below.

- Permanent vs Temporary - they were referring to if a disability was permanent vs temporary - this would fall under 'current disability'
- Benefit Entitlement would fall under 'Current benefits available/used'
-  Percentage used would fall under 'Current benefits available/used'
